### PR TITLE
Added information about Webhooks. 

### PR DESCRIPTION
**Pull request by Shy**

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -1 +1,8 @@
 require('dotenv').config();
+
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPassthroughCopy('css')
+  return {
```
Comment (stefanjudis) [inline .eleventy.js:5]: ❓Do you need this return statement here? I think you enable `passthrough` with the line above already?

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,6 @@ api-docs/
 _site/
 
 .DS_Store
+
+# Ignore env files
+.env
```
Comment (stefanjudis) [inline .gitignore:21]: 🧗‍♀️ Can we remove the checked in `.env` file? Having `.env` in ignore while shipping with an `.env` file could be a little confusing. 🙈

File: _includes/layout.liquid [hunk 3]
```diff
@@ -2,9 +2,20 @@
 <html lang='en'>
 
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{google-id}}"></script>
+    <script>
```
Comment (stefanjudis) [inline _includes/layout.liquid:7]: ⏳ Should we not load this if the optional google tracking id is not provided? :)

File: .gitignore (hunk 2 above)
Comment (Shy) [reply .gitignore:21]: I'll drop it from the ignore file. I added it so I wouldn't accidentally add the GA credentials. I want to ship with the .env file so folks can use my Contentful credentials and get a functional build straight away.

File: .eleventy.js (hunk 1 above)
Comment (Shy) [reply .eleventy.js:5]: Just tried it and I don't need it. I'll cut it.

File: _includes/layout.liquid (hunk 3 above)
Comment (Shy) [reply _includes/layout.liquid:8]: Fixed.

**Pull request merged**